src/lib/LED/devLED.cpp

```diff
@@ -65,6 +65,11 @@ static void initialize()
             pinMode(GPIO_PIN_LED_RED, OUTPUT);
             digitalWrite(GPIO_PIN_LED_RED, LOW ^ GPIO_LED_RED_INVERTED);
         #endif // GPIO_PIN_LED_RED
+        #if defined(TARGET_TX_IFLIGHT)
+            digitalWrite(GPIO_PIN_LED_GREEN, LOW);
+            digitalWrite(GPIO_PIN_LED_RED, HIGH);
+            digitalWrite(GPIO_PIN_LED_BLUE, LOW);
```

Comment:
Shouldn't all these have `^ GPIO_LED_XXX_INVERTED` on them, throughout the file.

Could we also change the `TARGET_TX_IFLIGHT` flag to something more meaningful that could be used by other targets as well. Perhaps we should specify that an RX or TX must specify the LEDs in order RED, GREEN, BLUE and we do different things based on how many LEDs are defined in the target.

Reply:
They could but are within the TARGET_TX_IFLIGHT define.  So it doesnt really matter.

Reply:
Yea, I do not like the `defined(TARGET_TX_IFLIGHT)`